src/aspire/basis/fpswf_2d.py

```diff
@@ -109,67 +108,30 @@ def _precomp(self):
 
     def evaluate_t(self, images):
         """
-        Evaluate coefficient vectors in PSWF basis using the fast method
+        Evaluate coefficient vectors in PSWF basis using the fast method.
 
-        :param images: coefficient array in the standard 2D coordinate basis
-            to be evaluated.
-        :return : The evaluation of the coefficient array in the PSWF basis.
+        :param images: Image stack in the standard 2D coordinate basis.
+        :return : Coefficient array in the PSWF basis.
         """
 
-        images = np.moveaxis(images, 0, -1)  # RCOPT
-
-        # start and finish are for the threads option in the future
-        images_shape = images.shape
-        start = 0
-
-        if len(images_shape) == 3:
-            # if we got several images
-            finish = images_shape[2]
-        else:
-            # else we got only one image
-            images_shape = images_shape + (1,)
-            images = images[..., np.newaxis]
-            finish = 1
-        images_disk = np.zeros(images.shape, dtype=images.dtype, order="F")
-        images_disk[self._disk_mask, :] = images[self._disk_mask, :]
-        nfft_res = self._compute_nfft_potts(images_disk, start, finish)
-        coefficients = self._pswf_integration(nfft_res)
-
-        return coefficients.T  # RCOPT
-
-    def evaluate(self, coefficients):
```

Comment:
So this function goes away because it's identical to `PSWFBasis2D.evaluate`. Have I understood that correctly?

Reply:
Yes, existing implementation was the same between the direct and fast.

Reply:
So…. not fast. Great.